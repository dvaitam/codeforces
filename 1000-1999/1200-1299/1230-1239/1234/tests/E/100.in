7 8
4 7 1 1 6 5 2 2
