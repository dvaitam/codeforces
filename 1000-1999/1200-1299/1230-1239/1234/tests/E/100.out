18 17 15 18 18 16 14
